src/mainboard/clevo/tgl-u/bootblock.c

```diff
@@ -34,5 +36,7 @@ void bootblock_mainboard_early_init(void)
 
 	pads = variant_early_gpio_table(&num);
 	gpio_configure_pads(pads, num);
+#ifdef DRIVER_NVIDIA_OPTIMUS
 	dgpu_power_enable(1);
```

Comment:
If this function is exposed by a header in nvidia driver, then
`if (CONFIG(DRIVER_NVIDIA_OPTIMUS))` would be enough

Reply:
i used `if (CONFIG(DRIVER_NVIDIA_OPTIMUS))` instead and removed all the `#ifdefs`